pontoon/urls.py

```diff
@@ -2,6 +2,11 @@
 from django.contrib import admin
 from django.contrib.auth.views import logout
 from django.views.generic import RedirectView, TemplateView
+from django.views.decorators.csrf import csrf_exempt
+from csp.decorators import csp_exempt
+from graphene_django.views import GraphQLView
```

Comment:
Nit: Please add blank lines in 5.5 and 7.5 to split Django, "other requirements" and pontoon imports into different sections. And move "other requirements" section above Django.

@adngdb Didn't we put the instructions for structuring import statements somewhere in our docs? Can't find.

Reply:
It's a part of PEP8: https://www.python.org/dev/peps/pep-0008/#imports

The only difference that we do is that we separate the framework imports from the rest of third party imports. I guess we could add a note about that in ``CONTRIBUTING.rst``?